SB2018/2018-02-14.md

```diff
@@ -0,0 +1,36 @@
+---
+to: All Teams
+subject: Tech day reminder - Saturday 17th Feb
+---
+
+Hi all,
+
+We've got another tech day this Saturday (17th Feb)!
+
+Tech days are a great opportunity for you to get robotics advice and help with technical issues from our volunteers. We may also have a tour of some university facilities as well.
+
+## Location
+
+We're currently trying to confirm a better venue for the tech day, but in the mean time, here are the details for our venue if we don't confirm the new one.
```

Comment:
I suggest that if we don't get a confirmation by EoD tomorrow (Thursday)  we don't use it, else it's making a change too close to the event

Reply:
Agree